plugins/automation/app/services/discourse_automation/destroy.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class DiscourseAutomation::Destroy
```

Comment:
I changed the name to `Destroy` instead of `DestroyAutomation` as we already have the namespace indicating this is `DiscourseAutomation`, it seemed redundant.

Reply:
It’s not necessarily a resource, the idea is more of using a namespace describing a business concept. But then, most of the model names are business concepts.